dependent/bundesgesundheitsblatt-gesundheitsforschung-gesundheitsschutz.csl

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="de-DE">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer-fachzeitschriften-medizin -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
-    <title>Bundesgesundheitsblatt - Gesundheitsforschung - Gesundheitsschutz (German)</title>
-    <title-short>Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz</title-short>
+    <title>Bundesgesundheitsblatt - Gesundheitsforschung - Gesundheitsschutz</title>
```

Comment:
Sure. But you agree that in this case, we actually shouldn't be using "Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz" as the short title, right, since it's the same as the full title?